src/PostgREST/App.hs

```diff
@@ -1,4 +1,4 @@
-{-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE FlexibleContexts, OverloadedStrings #-}
```

Comment:
I don't think we need this since the cabal file specifies OverloadedStrings as one of the default-extensions.